<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Radiologie statistiques</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Statistique de mois ****/******* :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>450</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Poumon</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>OS</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Abdomen simple</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>U.I.V</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Cholecystographie</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Estomac</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Echographie</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Fibroscopie</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>E.C.G</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>TOTAL</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Homme</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Famme</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Enfant</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TOTAL</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
